apis/core/v1alpha1/featureflagconfiguration_types.go

```diff
@@ -35,10 +35,17 @@ type FeatureFlagConfigurationSpec struct {
 	// +optional
 	// +nullable
 	SyncProvider *FeatureFlagSyncProvider `json:"syncProvider"`
+	// +optional
+	// +nullable
+	FlagDSpec *FlagDSpec `json:"flagDSpec"`
 	// FeatureFlagSpec is the json representation of the feature flag
 	FeatureFlagSpec string `json:"featureFlagSpec,omitempty"`
 }
 
+type FlagDSpec struct {
+	Port string `json:"port"`
```

Comment:
Is it work adding TLS-related configs here as well? Or should we just open a new issue for that?

Reply:
I think the latter, happy to do so